README.md

```diff
@@ -138,10 +140,16 @@ spec:
     metadata:
       labels:
         app: nginx
+      annotations:
+        osiris.deislabs.io/enabled: "true"
```

Comment:
@krancour I'm wondering if we shouldn't give this annotation a different name, to avoid confusion with the one on the deployment. This one is only used to inject the proxy, so maybe `osiris.deislabs.io/injectProxy: "true"` ?

I'm saying that because I have something else I'd like to add to osiris: a proxy-less mode. The proxy is only used to collect metrics, but we could imagine different implementations of the metrics collector, with one collecting metrics from a prometheus endpoint. For people already using prometheus, this would have the following benefits:
- complete control over how a request is counted, ie no need to use the `ignoredPaths`, and if needed requests can be ignored based on different input: user-agent, source IP, ...
- allow the use of another tool that inject a transparent proxy as a sidecar container, like a service mesh.
I imagine using a different metrics collector could be done by:
- not adding the `osiris.deislabs.io/injectProxy` annotation on the pods
- adding a new annotation on the deployment to enable the prometheus metrics collector, and specify the port, endpoint, metrics names, ...
I'll create a ticket to detail this proposal. so all that just to say that maybe we could use a more-specific name for this annotation ;-)

Reply:
New feature aside, I am on board with renaming this annotation (and for that matter, renaming `osiris.deislabs.io/enabled` in the other places it is used as well) with names that are more unambiguously indicative of the specific behaviors that are being added.

I do, however, think that should happen in a follow-up PR. In the even that renaming some annotations proves controversial, it will be easier to undo.